### ISSUE TITLE
[BUG] pt: Force loss is printed as positive value when no force data is available

### ISSUE THREAD

**Issue by njzjz** [deepmodeling/deepmd-kit#3483]
### Bug summary

Need to check whether it is a loss issue or just a printing issue.

### DeePMD-kit Version

v3.0.0a0-28-ged831c88-dirty

### TensorFlow Version

PT v2.2.0+cu121-g8ac9b20d4b0

### How did you download the software?

Build from source

### Input Files, Running Commands, Error Log, etc.

`rmse_f_val` and `rmse_f_trn` are still printed. Not sure whether they are being trained or not.

```
#  step      rmse_val    rmse_trn    rmse_e_val  rmse_e_trn    rmse_f_val  rmse_f_trn         lr
      0      7.41e+00    1.83e+00      5.01e-01    9.29e-01      2.32e-01    4.51e-03    1.0e-03
    100      2.98e-02    2.91e-02      3.75e-03    2.64e-03      9.10e-04    9.05e-04    1.0e-03
    200      1.45e-02    1.70e-02      3.21e-03    5.54e-03      3.92e-04    4.14e-04    1.0e-03
```

### Steps to Reproduce

Remove all `force.npy` from the training data.

```diff
diff --git a/examples/water/data/data_0/set.000/force.npy b/examples/water/data/data_0/set.000/force.npy
deleted file mode 100644
index 10b2ab83..00000000
Binary files a/examples/water/data/data_0/set.000/force.npy and /dev/null differ
diff --git a/examples/water/data/data_1/set.000/force.npy b/examples/water/data/data_1/set.000/force.npy
deleted file mode 100644
index dd8fccb4..00000000
Binary files a/examples/water/data/data_1/set.000/force.npy and /dev/null differ
diff --git a/examples/water/data/data_1/set.001/force.npy b/examples/water/data/data_1/set.001/force.npy
deleted file mode 100644
index 431e38cc..00000000
Binary files a/examples/water/data/data_1/set.001/force.npy and /dev/null differ
diff --git a/examples/water/data/data_2/set.000/force.npy b/examples/water/data/data_2/set.000/force.npy
deleted file mode 100644
index f1ad439e..00000000
Binary files a/examples/water/data/data_2/set.000/force.npy and /dev/null differ
diff --git a/examples/water/data/data_3/set.000/force.npy b/examples/water/data/data_3/set.000/force.npy
deleted file mode 100644
index e85711a4..00000000
Binary files a/examples/water/data/data_3/set.000/force.npy and /dev/null differ
```

Then run
```sh
cd examples/water/se_e2_a
dp --pt train input_torch.json
```

### Further Information, Files, and Links

- #3047
